x-pack/filebeat/input/netflow/testdata/golden/IPFIX-Barracuda-extended-uniflow-template-256.golden.json

```diff
@@ -0,0 +1,153 @@
+{
+  "test_name": "IPFIX Barracuda extended uniflow template 256",
+  "events": [
+    {
+      "Timestamp": "2018-04-18T08:16:47Z",
+      "Meta": null,
+      "Fields": {
```

Comment:
Ok, I understand that to review the key names, the object nested under `Fields` is the actual event. Above that is just test data structure. Correct?

Reply:
Yes, although you should review the ECS mappings in `convert.go`, as events may not contain all the possible ECS mappings.